drivers/volume/portworx/portworx.go

```diff
@@ -1623,6 +1638,50 @@ func (d *portworx) ValidatePureFaCreateOptions(volumeName string, FStype string,
 	return nil
 }
 
+// ValidatePureRawBlockVolumes validates rawblock pure volumes
+func (d *portworx) ValidatePureRawBlockVolumes(volumeName string, pvcObj *corev1.PersistentVolumeClaim) error {
```

Comment:
This method is not needed if all these commands are part of application spec itself and either ValidateContextForPureVolumesSDK or ValidateContext can be use to validate the application.

Reply:
Done.